training/pyproject.toml

```diff
@@ -129,7 +129,7 @@ lint.select = [
   "C4",     # Comprehensions
   "C90",    # Mccabe complexity
   "COM",    # Commas
-  "CPY",    # Copyright
+  # "CPY",    # Copyright
```

Comment:
Is this a local change or was this giving you any errors?

Reply:
I don't remember honestly.